README.md

```diff
@@ -1,32 +1,45 @@
+
+</p>
+
+
 Requiem is a Minecraft (Java Edition) mod aiming at removing the continuity break resulting from death,
 as well as adding new mechanics centered around soul dissociation and body possession.
 
-More information is available on the Ladysnake Website: https://ladysnake.github.io/wiki/requiem.
+More information is available on the Ladysnake Website: <https://ladysnake.github.io/wiki/requiem>.
 
 ## Installation
 
 ### For players
 
-- Running the mod will require Fabric and Fabric API. For more information on how to use Fabric, [please visit the official website](https://fabricmc.net/). To download Fabric API, [please visit the official CurseForge page](https://minecraft.curseforge.com/projects/fabric).
-- If Fabric is installed on your Minecraft Java copy, you can download Requiem from [CurseForge](https://minecraft.curseforge.com/projects/requiem), [Github Releases](https://github.com/Ladysnake/Requiem/releases), or [Modrinth](https://modrinth.com/mod/requiem). After that, copy the jar file obtained, as well as the download Fabric API jar  file, in the mods folder located in you Minecraft installation or Fabric Server folder. Please make sure to have the latest version, as we will  not accept issues regarding older versions.
-- Once the previous steps have been completed, boot up your game, and  upon your first death, Requiem will asks you whether you'd like to  enable the modified death system or not through an interactive dialogue. Be careful, because after choosing, the dialogue won't be prompted a  second time, and you will have to use a special item to change that.
+*   Running the mod will require Quilt and Quilted Fabric API. For more information on how to use Quilt, [please visit the official website](https://quiltmc.org/). To download Quilted Fabric API, please visit the official [CurseForge](https://www.curseforge.com/minecraft/mc-mods/qsl) or [Modrinth](https://modrinth.com/mod/qsl) page.
+
+*   If Quilt is installed on your Minecraft Java copy, you can download Requiem from [CurseForge](https://minecraft.curseforge.com/projects/requiem), [Github Releases](https://github.com/Ladysnake/Requiem/releases), or [Modrinth](https://modrinth.com/mod/requiem). After that, copy the jar file obtained, as well as the Quilted Fabric API jar file in the mods folder located in you Minecraft installation or your Server's Mods folder. Please make sure to have the latest version, as we will not accept issues regarding older versions.
+
+*   Once the previous steps have been completed, boot up your game, and  upon your first death, Requiem will asks you whether you'd like to  enable the modified death system or not through an interactive dialogue. Be careful, because after choosing, the dialogue won't be prompted a second time, and you will have to use a special item to change that.

```

Comment:
Why change the list style ?

Reply:
Same as above, didn't actively change this. Suppose that Milkdown changed it, feel free to undo it.